proto/autokitteh/store/v1/svc.proto

```diff
@@ -11,9 +11,8 @@ service StoreService {
 }
 
 message GetRequest {
-  string env_id = 1;
-  string project_id = 2;
-  repeated string keys = 3 [(buf.validate.field).repeated.items.string.min_len = 1];
+  string project_id = 1;
```

Comment:
I'm fine with this, but to be clear you're committing multiple violations here at a point in time that AK is no longer in its "early days":
- Delete a field instead of deprecating it
- Change a field ID